<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List href="sample_files/filelist.xml">
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Rashi Verenker</o:Author>
  <o:Template>Normal</o:Template>
  <o:LastAuthor>Rashi Verenker</o:LastAuthor>
  <o:Revision>2</o:Revision>
  <o:TotalTime>4</o:TotalTime>
  <o:Created>2021-05-21T10:07:00Z</o:Created>
  <o:LastSaved>2021-05-21T10:07:00Z</o:LastSaved>
  <o:Pages>15</o:Pages>
  <o:Words>13518</o:Words>
  <o:Characters>77057</o:Characters>
  <o:Lines>642</o:Lines>
  <o:Paragraphs>180</o:Paragraphs>
  <o:CharactersWithSpaces>90395</o:CharactersWithSpaces>
  <o:Version>16.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="sample_files/themedata.thmx">
<link rel=colorSchemeMapping href="sample_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:SpellingState>Clean</w:SpellingState>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves/>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-IN</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"
  DefSemiHidden="false" DefQFormat="false" DefPriority="99"
  LatentStyleCount="376">
  <w:LsdException Locked="false" Priority="0" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 9"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="header"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footer"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index heading"/>
  <w:LsdException Locked="false" Priority="35" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of figures"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope return"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="line number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="page number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of authorities"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="macro"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="toa heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 5"/>
  <w:LsdException Locked="false" Priority="10" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Closing"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Signature"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="true"
   UnhideWhenUsed="true" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Message Header"/>
  <w:LsdException Locked="false" Priority="11" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Salutation"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Date"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Note Heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Block Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="FollowedHyperlink"/>
  <w:LsdException Locked="false" Priority="22" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Document Map"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Plain Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="E-mail Signature"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Top of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Bottom of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal (Web)"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Acronym"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Cite"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Code"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Definition"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Keyboard"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Preformatted"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Sample"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Typewriter"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Variable"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Table"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation subject"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="No List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Contemporary"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Elegant"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Professional"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Balloon Text"/>
  <w:LsdException Locked="false" Priority="39" Name="Table Grid"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Theme"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" QFormat="true"
   Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" QFormat="true"
   Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" QFormat="true"
   Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" QFormat="true"
   Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" QFormat="true"
   Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" QFormat="true"
   Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" SemiHidden="true"
   UnhideWhenUsed="true" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/>
  <w:LsdException Locked="false" Priority="41" Name="Plain Table 1"/>
  <w:LsdException Locked="false" Priority="42" Name="Plain Table 2"/>
  <w:LsdException Locked="false" Priority="43" Name="Plain Table 3"/>
  <w:LsdException Locked="false" Priority="44" Name="Plain Table 4"/>
  <w:LsdException Locked="false" Priority="45" Name="Plain Table 5"/>
  <w:LsdException Locked="false" Priority="40" Name="Grid Table Light"/>
  <w:LsdException Locked="false" Priority="46" Name="Grid Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="Grid Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="Grid Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="46" Name="List Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="List Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="List Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hashtag"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Unresolved Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Link"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-536869121 1107305727 33554432 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-469750017 -1073732485 9 0 511 0;}
@font-face
	{font-family:Georgia;
	panose-1:2 4 5 2 5 4 5 2 3 3;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:647 0 0 0 159 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
.MsoPapDefault
	{mso-style-type:export-only;
	margin-bottom:8.0pt;
	line-height:107%;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:1.0cm 1.0cm 1.0cm 42.55pt;
	mso-header-margin:35.4pt;
	mso-footer-margin:35.4pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:814178735;
	mso-list-type:hybrid;
	mso-list-template-ids:1874896156 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	margin-left:32.15pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l0:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l0:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l0:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l0:level5
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l0:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l0:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l0:level8
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l0:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l1
	{mso-list-id:1275865170;
	mso-list-type:hybrid;
	mso-list-template-ids:919524984 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l1:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l1:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l1:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l1:level5
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l1:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l1:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l1:level8
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l1:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l2
	{mso-list-id:1489637204;
	mso-list-type:hybrid;
	mso-list-template-ids:2059585382 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653 1074331649 1074331651 1074331653;}
@list l2:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l2:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l2:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l2:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l2:level5
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l2:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l2:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l2:level8
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:"Courier New";}
@list l2:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin-top:0cm;
	mso-para-margin-right:0cm;
	mso-para-margin-bottom:8.0pt;
	mso-para-margin-left:0cm;
	line-height:107%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
table.MsoTableGrid
	{mso-style-name:"Table Grid";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-priority:39;
	mso-style-unhide:no;
	border:solid windowtext 1.0pt;
	mso-border-alt:solid windowtext .5pt;
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-border-insideh:.5pt solid windowtext;
	mso-border-insidev:.5pt solid windowtext;
	mso-para-margin:0cm;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-IN style='tab-interval:36.0pt;word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
line-height:115%'><a name="_Hlk72503474"><b style='mso-bidi-font-weight:normal'><span
lang=EN-US style='font-size:16.0pt;line-height:115%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US'>ABSTRACT<o:p></o:p></span></b></a></p>

<p class=MsoNormal style='margin-bottom:0cm;line-height:115%'><span
style='mso-bookmark:_Hlk72503474'><span lang=EN-US style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:bold'><o:p>&nbsp;</o:p></span></span></p>

<p class=MsoNormal style='margin-bottom:0cm;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:bold'>Colorectal
cancer is a deadly disease which affects hundreds and thousands of individuals each
year. There is a need to understand the underlying genetic mechanisms involved
in order to develop effective methods for the diagnosis and treatment, and,
eventually, cure colorectal cancer. <a name="_Hlk72385312">The purpose of this
study was to build a transcriptome profile for CRC and identify the key
regulatory genes involved</a>. This study also aimed at finding the molecular
signatures using the genes identified and understand the molecular mechanisms
that played a role in linking the genotype to phenotype. Significant genes were
identified by mapping datasets to a reference genome and based on the counts
table generated, DESeq2 was used for gene analysis. Volcano plots exhibit the expression
of up-regulated and downregulated genes. <a name="_Hlk72389978">469
up-regulated genes and 162 downregulated </a>genes were identified. Clustering
analysis was done to find networks and top ten significant genes were used to
find molecular signatures. The shortest paths among the target (seed) and
source (significant genes) were found. Variant calling was done where somatic
and germline variants were identified as well as variants which were affected
by loss of heterozygosity. Variants and genes affected by these variants were
found and they were annotated according to general human genetic and
cancer-specific databases.<o:p></o:p></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></span></p>

<b><span style='font-size:16.0pt;line-height:107%;font-family:"Times New Roman",serif;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-ansi-language:
EN-IN;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span></b>

<p class=MsoNormal><span style='mso-bookmark:_Hlk72503474'><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><b><span style='font-size:16.0pt;line-height:
150%;font-family:"Times New Roman",serif'>Introduction<o:p></o:p></span></b></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Cancer
is a deadly disease which is on the rise. Colorectal cancer (CRC) is that which
occurs in the colon and the rectum. Hundreds of thousands of individuals are
diagnosed each year and due to this it is ranked high under basic cancers. Compared
to males, females are more commonly affected and this makes it second common cancer
in women and third in men </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Stewart&quot;,&quot;given&quot;:&quot;BW&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kleihues&quot;,&quot;given&quot;:&quot;P&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2003&quot;]]},&quot;title&quot;:&quot;World
cancer report&quot;,&quot;type&quot;:&quot;article-journal&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=ef1b404d-3fb2-3cbf-8bd0-4b331f64c4ba&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Stewart
and Kleihues 2003)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Stewart
and Kleihues 2003)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Stewart
and Kleihues 2003)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Stewart
and Kleihues 2003)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. In the Western countries, its
prevalence is much higher than that in India. In terms of mortality causes, it
is ranked seventh in India. Recent study conducted showed that as many as 28,000
cases were diagnosed with mortality of around 21,000. It is estimated that
there could be approximately 50,000 active cases in India as of now. Normally
CRC affects individuals over the age of 50 however there has been an increase
in the number of cases in the younger population. About 32% of the individuals
are younger than 45.<o:p></o:p></span></span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0 width=472
 style='width:354.35pt;margin-left:77.6pt;border-collapse:collapse;border:none;
 mso-border-alt:solid windowtext .5pt;mso-yfti-tbllook:1184;mso-padding-alt:
 0cm 5.4pt 0cm 5.4pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Age
  of individuals diagnosed with CRC<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Percentage<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>&lt;20<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>4.29<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>20-29<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>13.30<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:3'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>30-39<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>21.45<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:4'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>40-49<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>24.89<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:5'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>50-59<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>23.60<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:6'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>60-69<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>9.87<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:7'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>&gt;70<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>2.57<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
 <tr style='mso-yfti-irow:8;mso-yfti-lastrow:yes'>
  <td width=283 valign=top style='width:212.3pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Total<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
  <td width=189 valign=top style='width:142.05pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;text-align:center;
  line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
  style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>100<o:p></o:p></span></span></p>
  </td>
  <span style='mso-bookmark:_Hlk72503474'></span>
 </tr>
</table>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>Table 1: Percentage of individuals
affected due to CRC according to age<o:p></o:p></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>A
miniscule fraction of CRCs is caused due to mutations in the inherited genes. APC
is involved in tumor suppression and plays a vital role in keeping cell growth
under check. Irregularities in genes involved in repair, such as PMS2, MLH1,
MLH2 AND MLH6, could lead to oversight while regulating growth related genes
and may prove to be a key factor in cancer formation </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1056/NEJMra012242&quot;,&quot;ISSN&quot;:&quot;0028-4793&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lynch&quot;,&quot;given&quot;:&quot;Henry
T.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;la
Chapelle&quot;,&quot;given&quot;:&quot;Albert&quot;,&quot;non-dropping-particle&quot;:&quot;de&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;New
England Journal of
Medicine&quot;,&quot;editor&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Guttmacher&quot;,&quot;given&quot;:&quot;Alan
E.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Collins&quot;,&quot;given&quot;:&quot;Francis
S.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2003&quot;,&quot;3&quot;,&quot;6&quot;]]},&quot;page&quot;:&quot;919-932&quot;,&quot;publisher&quot;:&quot;N
Engl J Med&quot;,&quot;title&quot;:&quot;Hereditary Colorectal
Cancer&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;348&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=5a22fae1-b6fd-3b3f-9535-5ff9c9ba6275&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Lynch
and de la Chapelle 2003)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Lynch
and de la Chapelle
2003)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Lynch and de la
Chapelle
2003)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Lynch
and de la Chapelle 2003)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Similarly, STK11, which encodes for
serine, plays a role in suppression. MUTYH manages the analysis of cell and
variation in this gene can cause rampant, unchecked division.<o:p></o:p></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Three
notable pathways affect play a role in CRC </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION {&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/nm1087&quot;,&quot;ISSN&quot;:&quot;10788956&quot;,&quot;PMID&quot;:&quot;15286780&quot;,&quot;abstract&quot;:&quot;The
revolution in cancer research can be summed up in a single sentence: cancer is,
in essence, a genetic disease. In the last decade, many important genes
responsible for the genesis of various cancers have been discovered, their
mutations precisely identified, and the pathways through which they act
characterized. The purposes of this review are to highlight examples of
progress in these areas, indicate where knowledge is scarce and point out
fertile grounds for future
investigation.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Vogelstein&quot;,&quot;given&quot;:&quot;Bert&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kinzler&quot;,&quot;given&quot;:&quot;Kenneth
W.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Medicine&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2004&quot;,&quot;8&quot;,&quot;30&quot;]]},&quot;page&quot;:&quot;789-799&quot;,&quot;publisher&quot;:&quot;Nature
Publishing Group&quot;,&quot;title&quot;:&quot;Cancer genes and the pathways
they
control&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=ffb8fd21-ed85-3e39-ae71-d25e5176e045&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Vogelstein
and Kinzler
2004)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Vogelstein and
Kinzler 2004)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Vogelstein
and Kinzler
2004)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Vogelstein
and Kinzler 2004)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. The first pathway is Chromosomal Instability
(CIN). This is a traditional pathway and is successful in explaining large number
of cancers </span></span><!--[if supportFields]><span style='mso-bookmark:_Hlk72503474'></span><span
style='mso-element:field-begin;mso-field-lock:yes'></span><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1053/j.gastro.2008.07.076&quot;,&quot;ISSN&quot;:&quot;00165085&quot;,&quot;PMID&quot;:&quot;18773902&quot;,&quot;abstract&quot;:&quot;Colorectal
cancer arises as a consequence of the accumulation of genetic alterations (gene
mutations, gene amplification, and so on) and epigenetic alterations (aberrant
DNA methylation, chromatin modifications, and so on) that transform colonic
epithelial cells into colon adenocarcinoma cells. The loss of genomic stability
and resulting gene alterations are key molecular pathogenic steps that occur
early in tumorigenesis; they permit the acquisition of a sufficient number of
alterations in tumor suppressor genes and oncogenes that transform cells and
promote tumor progression. Two predominant forms of genomic instability that
have been identified in colon cancer are microsatellite instability and
chromosome instability. Substantial progress has been made to identify causes
of chromosomal instability in colorectal cells and to determine the effects of
the different forms of genomic instability on the biological and clinical
behavior of colon tumors. In addition to genomic instability, epigenetic
instability results in the aberrant methylation of tumor suppressor genes.
Determining the causes and roles of genomic and epigenomic instability in colon
tumor formation has the potential to yield more effective prevention strategies
and therapeutics for patients with colorectal cancer.  2008 AGA
Institute.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Grady&quot;,&quot;given&quot;:&quot;William
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Carethers&quot;,&quot;given&quot;:&quot;John
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Gastroenterology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2008&quot;]]},&quot;page&quot;:&quot;1079-1099&quot;,&quot;publisher&quot;:&quot;W.B.
Saunders&quot;,&quot;title&quot;:&quot;Genomic and Epigenetic Instability in
Colorectal Cancer
Pathogenesis&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;135&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=04a09ccd-925d-3ef2-abd4-ced5da6f1c70&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Grady
and Carethers 2008)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Grady
and Carethers 2008)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Grady
and Carethers
2008)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Grady
and Carethers 2008)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. It describes the uncertainties in
chromosomes and their number which eventually leads to tumor and loss of one
copy of a gene. Working principle covers the revision in chromosome division
and <span class=SpellE>irregulation</span> in telomere. This in turn affects
the working capacity of genes such as <i>APC</i>,&nbsp;<i>KRAS</i>,&nbsp;<i>PI3K</i>&nbsp;and&nbsp;<i>TP53.</i>
Variation in TP53 causes uninterrupted entry in cell cycle </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1053/j.gastro.2009.12.065&quot;,&quot;ISSN&quot;:&quot;00165085&quot;,&quot;PMID&quot;:&quot;20420946&quot;,&quot;abstract&quot;:&quot;The
acquisition of genomic instability is a crucial feature in tumor development
and there are at least 3 distinct pathways in colorectal cancer pathogenesis:
the chromosomal instability (CIN), microsatellite instability, and CpG island
methylator phenotype pathways. Most cases of colorectal cancer arise through
the CIN pathway, which is characterized by widespread imbalances in chromosome
number (aneuploidy) and loss of heterozygosity. It can result from defects in
chromosomal segregation, telomere stability, and the DNA damage response,
although the full complement of genes underlying CIN remains incompletely
described. Coupled with the karyotypic abnormalities observed in CIN tumors are
the accumulation of a characteristic set of mutations in specific tumor
suppressor genes and oncogenes that activate pathways critical for colorectal
cancer initiation and progression. Whether CIN creates the appropriate milieu
for the accumulation of these mutations or vice versa remains a provocative and
unanswered question. The goal of this review is to provide an updated
perspective on the mechanisms that lead to CIN and the key mutations that are
acquired in this pathway.  2010 AGA
Institute.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Pino&quot;,&quot;given&quot;:&quot;Maria
S.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Chung&quot;,&quot;given&quot;:&quot;Daniel
C.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Gastroenterology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2010&quot;]]},&quot;page&quot;:&quot;2059-2072&quot;,&quot;publisher&quot;:&quot;W.B.
Saunders&quot;,&quot;title&quot;:&quot;The Chromosomal Instability Pathway in
Colon Cancer&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;138&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=27604f54-b838-3a72-8441-6746452bcd60&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Pino
and Chung 2010)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Pino and
Chung 2010)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Pino and Chung
2010)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Pino
and Chung 2010)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Microsatellite instability pathway
comes into play when DNA repair is no longer effective and this causes overexpression
of phenotype. DNA repair ability is severely compromised and variations end up
compiling which eventually affects coding and non-coding regions in
microsatellites. The reading frame gets affected and tumors form. These are primarily
diploid. Genes affected are MLH1, MSH2, MSH6, PMS1 and PMS2 </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1053/j.gastro.2009.12.064&quot;,&quot;ISSN&quot;:&quot;00165085&quot;,&quot;PMID&quot;:&quot;20420947&quot;,&quot;abstract&quot;:&quot;Microsatellite
instability (MSI) is a hypermutable phenotype caused by the loss of DNA
mismatch repair activity. MSI is detected in about 15% of all colorectal
cancers; 3% are of these are associated with Lynch syndrome and the other 12%
are caused by sporadic, acquired hypermethylation of the promoter of the MLH1
gene, which occurs in tumors with the CpG island methylator phenotype.
Colorectal tumors with MSI have distinctive features, including a tendency to
arise in the proximal colon, lymphocytic infiltrate, and a poorly
differentiated, mucinous or signet ring appearance. They have a slightly better
prognosis than colorectal tumors without MSI and do not have the same response
to chemotherapeutics. Discovery of MSI in colorectal tumors has increased
awareness of the diversity of colorectal cancers and implications for
specialized management of patients.  2010 AGA
Institute.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Boland&quot;,&quot;given&quot;:&quot;C.
Richard&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Goel&quot;,&quot;given&quot;:&quot;Ajay&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Gastroenterology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2010&quot;]]},&quot;publisher&quot;:&quot;W.B.
Saunders&quot;,&quot;title&quot;:&quot;Microsatellite Instability in Colorectal
Cancer&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;138&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=04a7b794-aeca-3c8c-88b3-9c408938025c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Boland
and Goel 2010)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Boland and
Goel 2010)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Boland and Goel
2010)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Boland
and Goel 2010)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Epigenetic instability plays a
vital role in CpG islands. CIMP tumors cause excessive methylation and this
causes silencing and eventually loss of protein expression. <o:p></o:p></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Genetics
and epigenetics together help in advancement of CRC and there is increased
methylation compared to variations </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/nrgastro.2011.173&quot;,&quot;ISSN&quot;:&quot;17595045&quot;,&quot;PMID&quot;:&quot;22009203&quot;,&quot;abstract&quot;:&quot;Colorectal
cancer (CRC) is a leading cause of cancer deaths worldwide. It results from an
accumulation of genetic and epigenetic changes in colon epithelial cells, which
transforms them into adenocarcinomas. Over the past decade, major advances have
been made in understanding cancer epigenetics, particularly regarding aberrant
DNA methylation. Assessment of the colon cancer epigenome has revealed that
virtually all CRCs have aberrantly methylated genes and that the average CRC
methylome has hundreds to thousands of abnormally methylated genes. As with
gene mutations in the cancer genome, a subset of these methylated genes, called
driver genes, is presumed to have a functional role in CRC. The assessment of
methylated genes in CRCs has also revealed a unique molecular subgroup of CRCs
called CpG island methylator phenotype (CIMP) cancers; these tumors have a
particularly high frequency of methylated genes. These advances in our
understanding of aberrant methylation in CRC have led to epigenetic alterations
being developed as clinical biomarkers for diagnostic, prognostic and therapeutic
applications. Progress in this field suggests that these epigenetic alterations
will be commonly used in the near future to direct the prevention and treatment
of CRC.  2012 Adis Data Information BV. All rights
reserved.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lao&quot;,&quot;given&quot;:&quot;Victoria
Valinluck&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Grady&quot;,&quot;given&quot;:&quot;William
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Reviews Gastroenterology and
Hepatology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,&quot;11&quot;]]},&quot;page&quot;:&quot;686-700&quot;,&quot;publisher&quot;:&quot;Nat
Rev Gastroenterol Hepatol&quot;,&quot;title&quot;:&quot;Epigenetics and
colorectal
cancer&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;8&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=861d61a1-1015-3d33-8580-f4603fb26e0c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Lao
and Grady 2011)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Lao and
Grady 2011)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Lao and Grady
2011)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Lao
and Grady 2011)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Example of this is BRAF mutations along
with microsatellite instability CIMP tumors </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/ng1834&quot;,&quot;ISSN&quot;:&quot;10614036&quot;,&quot;PMID&quot;:&quot;16804544&quot;,&quot;abstract&quot;:&quot;Aberrant
DNA methylation of CpG islands has been widely observed in human colorectal
tumors and is associated with gene silencing when it occurs in promoter areas.
A subset of colorectal tumors has an exceptionally high frequency of
methylation of some CpG islands, leading to the suggestion of a distinct trait
referred to as 'CpG island methylator phenotype', or 'CIMP'. However, the
existence of CIMP has been challenged. To resolve this continuing controversy,
we conducted a systematic, stepwise screen of 195 CpG island methylation
markers using MethyLight technology, involving 295 primary human colorectal
tumors and 16,785 separate quantitative analyses. We found that CIMP-positive
(CIMP+) tumors convincingly represent a distinct subset, encompassing almost
all cases of tumors with BRAF mutation (odds ratio = 203). Sporadic cases of mismatch
repair deficiency occur almost exclusively as a consequence of CIMP-associated
methylation of MLH1. We propose a robust new marker panel to classify CIMP+
tumors.  2006 Nature Publishing
Group.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Weisenberger&quot;,&quot;given&quot;:&quot;Daniel
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Siegmund&quot;,&quot;given&quot;:&quot;Kimberly
D.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Campan&quot;,&quot;given&quot;:&quot;Mihaela&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Young&quot;,&quot;given&quot;:&quot;Joanne&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Long&quot;,&quot;given&quot;:&quot;Tiffany
I.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Faasse&quot;,&quot;given&quot;:&quot;Mark
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kang&quot;,&quot;given&quot;:&quot;Gyeong
Hoon&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Widschwendter&quot;,&quot;given&quot;:&quot;Martin&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Weener&quot;,&quot;given&quot;:&quot;Deborah&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Buchanan&quot;,&quot;given&quot;:&quot;Daniel&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Koh&quot;,&quot;given&quot;:&quot;Hoey&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Simms&quot;,&quot;given&quot;:&quot;Lisa&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Barker&quot;,&quot;given&quot;:&quot;Melissa&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Leggett&quot;,&quot;given&quot;:&quot;Barbara&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Levine&quot;,&quot;given&quot;:&quot;Joan&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kim&quot;,&quot;given&quot;:&quot;Myungjin&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;French&quot;,&quot;given&quot;:&quot;Amy
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Thibodeau&quot;,&quot;given&quot;:&quot;Stephen
N.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Jass&quot;,&quot;given&quot;:&quot;Jeremy&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Haile&quot;,&quot;given&quot;:&quot;Robert&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Laird&quot;,&quot;given&quot;:&quot;Peter
W.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Genetics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2006&quot;,&quot;7&quot;]]},&quot;page&quot;:&quot;787-793&quot;,&quot;publisher&quot;:&quot;Nat
Genet&quot;,&quot;title&quot;:&quot;CpG island methylator phenotype underlies
sporadic microsatellite instability and is tightly associated with BRAF
mutation in colorectal
cancer&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;38&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=7592b3b1-9129-336c-975c-791f042c3ccd&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Weisenberger
et al. 2006)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Weisenberger
et al. 2006)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Weisenberger
et al.
2006)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Weisenberger
et al. 2006)</span></span></span><!--[if supportFields]><span style='mso-bookmark:
_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. <o:p></o:p></span></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>Gene expression profiling is the
comparison between the expression in tumor and normal tissue samples. This is
essential as it provides valuable information and aids in prognosis </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72503474'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72503474'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/sj.onc.1207262&quot;,&quot;ISSN&quot;:&quot;09509232&quot;,&quot;PMID&quot;:&quot;14973550&quot;,&quot;abstract&quot;:&quot;Different
diagnostic and prognostic groups of colorectal carcinoma (CRC) have been
defined. However, accurate diagnosis and prediction of survival are sometimes
difficult. Gene expression profiling might improve these classifications and
bring new insights into underlying molecular mechanisms. We profiled 50
cancerous and noncancerous colon tissues using DNA microarrrays consisting of </span></span><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Cambria Math",serif;mso-bidi-font-family:"Cambria Math"'>&#8764;</span></span><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>8000 spotted human cDNA. Global
hierarchical clustering was to some extent able to distinguish clinically
relevant subgroups, normal versus cancer tissues and metastatic versus
nonmetastatic tumours. Supervised analyses improved these segregations by
identifying sets of genes that discriminated between normal and tumour tissues,
tumours associated or not with lymph node invasion or genetic instability, and
tumours from the right or left colon. A similar approach identified a gene set
that divided patients with significantly different 5-year survival (100% in one
group and 40% in the other group; P =0.005). Discriminator genes were
associated with various cellular processes. An immunohistochemical study on 382
tumour and normal samples deposited onto a tissue microarray subsequently
validated the upregulation of NM23 in CRC and a downregulation in poor
prognosis tumours. These results suggest that microarrays may provide means to
improve the classification of CRC, provide new potential targets against
carcinogenesis and new diagnostic and/or prognostic markers and therapeutic
targets.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bertucci&quot;,&quot;given&quot;:&quot;Franois&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Salas&quot;,&quot;given&quot;:&quot;Sbastien&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Eysteries&quot;,&quot;given&quot;:&quot;Sverine&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Nasser&quot;,&quot;given&quot;:&quot;Valry&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Finetti&quot;,&quot;given&quot;:&quot;Pascal&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ginestier&quot;,&quot;given&quot;:&quot;Christophe&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Charafe-Jauffret&quot;,&quot;given&quot;:&quot;Emmanuelle&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Loriod&quot;,&quot;given&quot;:&quot;Batrice&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bachelart&quot;,&quot;given&quot;:&quot;Loc&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Montfort&quot;,&quot;given&quot;:&quot;Jrme&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Victorero&quot;,&quot;given&quot;:&quot;Genevive&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Viret&quot;,&quot;given&quot;:&quot;Frdric&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ollendorff&quot;,&quot;given&quot;:&quot;Vincent&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Fert&quot;,&quot;given&quot;:&quot;Vincent&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Giovaninni&quot;,&quot;given&quot;:&quot;Marc&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Delpero&quot;,&quot;given&quot;:&quot;Jean
Robert&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Nguyen&quot;,&quot;given&quot;:&quot;Catherine&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Viens&quot;,&quot;given&quot;:&quot;Patrice&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Monges&quot;,&quot;given&quot;:&quot;Genevive&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Birnbaum&quot;,&quot;given&quot;:&quot;Daniel&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Houlgatte&quot;,&quot;given&quot;:&quot;Rmi&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Oncogene&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2004&quot;,&quot;2&quot;,&quot;19&quot;]]},&quot;page&quot;:&quot;1377-1391&quot;,&quot;publisher&quot;:&quot;Oncogene&quot;,&quot;title&quot;:&quot;Gene
expression profiling of colon cancer by DNA microarrays and correlation with
histoclinical parameters&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;23&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=6de7e5cf-78d1-3eec-9863-2c82bbdff2ac&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Bertucci
et al. 2004)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Bertucci et
al. 2004)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Bertucci et al.
2004)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Bertucci
et al. 2004)</span></span></span><!--[if supportFields]><span style='mso-bookmark:
_Hlk72503474'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Advancements have been done in
these assays and advantages for patients as well as society have been highlighted.
However, there isnt enough proof to validate possible benefits to chemotherapy.
Further studies and exploration <span class=GramE>is</span> needed to validate
them clinically.<o:p></o:p></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72503474'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'>Objectives</span></b></span><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><o:p></o:p></span></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>To perform Differential Gene analysis
using Galaxy and find the up down regulated genes.<o:p></o:p></span></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>To find the molecular signatures
using <span class=SpellE>cytoscape</span>, obtaining clusters to identify
disease phenotype regulators and find out the key regulatory genes.<o:p></o:p></span></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>Perform Variant Calling to understand
genotype- phenotype relationship. <o:p></o:p></span></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='mso-bookmark:_Hlk72503474'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>Introduce a new dimension to further
aid in diagnosis, drug discovery or maybe assist in giving rise to novel
strategies for further analysis</span></span><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:16.0pt;
line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:16.0pt;
line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:16.0pt;
line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:16.0pt;
line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:16.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:16.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'>Review
of Literature<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><a name="_Hlk72500077"><b><span
style='font-size:14.0pt;line-height:150%;font-family:"Times New Roman",serif'>Molecular
based diagnostic methods<o:p></o:p></span></b></a></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:_Hlk72500077'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Colorectal
Cancer is a deadly disease and the cause of countless deaths. The microbial
organisms in the gut of an individual play a role in forming carcinogenic
colonies by following a pathway linked to inflammation </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72500077'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.3390/ijms18010197&quot;,&quot;ISSN&quot;:&quot;14220067&quot;,&quot;PMID&quot;:&quot;28106826&quot;,&quot;abstract&quot;:&quot;Colorectal
cancer (CRC) is the third most common cancer and the fourth most common cause
of cancer-related death. Most cases of CRC are detected inWestern countries,
with its incidence increasing year by year. The probability of suffering from
colorectal cancer is about 4%5% and the risk for developing CRC is associated
with personal features or habits such as age, chronic disease history and
lifestyle. In this context, the gut microbiota has a relevant role, and
dysbiosis situations can induce colonic carcinogenesis through a chronic
inflammation mechanism. Some of the bacteria responsible for this multiphase
process include Fusobacterium spp, Bacteroides fragilis and enteropathogenic
Escherichia coli. CRC is caused by mutations that target oncogenes, tumour
suppressor genes and genes related to DNA repair mechanisms. Depending on the
origin of the mutation, colorectal carcinomas can be classified as sporadic
(70%); inherited (5%) and familial (25%). The pathogenic mechanisms leading to
this situation can be included in three types, namely chromosomal instability
(CIN), microsatellite instability (MSI) and CpG island methylator phenotype
(CIMP). Within these types of CRC, common mutations, chromosomal changes and
translocations have been reported to affect important pathways (WNT, MAPK/PI3K,
TGF-&#946;, TP53), and mutations; in particular, genes such as c-MYC, KRAS,
BRAF, PIK3CA, PTEN, SMAD2 and SMAD4 can be used as predictive markers for
patient outcome. In addition to gene mutations, alterations in ncRNAs, such as
lncRNA or miRNA, can also contribute to different steps of the carcinogenesis
process and have a predictive value when used as biomarkers. In consequence,
different panels of genes and mRNA are being developed to improve prognosis and
treatment selection. The choice of first-line treatment in CRC follows a
multimodal approach based on tumour-related characteristics and usually
comprises surgical resection followed by chemotherapy combined with monoclonal
antibodies or proteins against vascular endothelial growth factor (VEGF) and
epidermal growth receptor (EGFR). Besides traditional chemotherapy, alternative
therapies (such as agarose tumour macrobeads, anti-inflammatory drugs,
probiotics, and gold-based drugs) are currently being studied to increase
treatment effectiveness and reduce side
effects.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Mrmol&quot;,&quot;given&quot;:&quot;Ins&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Snchez-de-Diego&quot;,&quot;given&quot;:&quot;Cristina&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dieste&quot;,&quot;given&quot;:&quot;Alberto
Pradilla&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Cerrada&quot;,&quot;given&quot;:&quot;Elena&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Yoldi&quot;,&quot;given&quot;:&quot;Mara
Jess Rodriguez&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;International
Journal of Molecular Sciences&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;,&quot;1&quot;,&quot;19&quot;]]},&quot;page&quot;:&quot;197&quot;,&quot;publisher&quot;:&quot;MDPI
AG&quot;,&quot;title&quot;:&quot;Colorectal carcinoma: A general overview and
future perspectives in colorectal
cancer&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;18&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=9e7b8907-0a7d-322d-80cd-496ad953e685&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Mrmol
et al. 2017)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Mrmol et al.
2017)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Mrmol et al.
2017)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Mrmol
et al. 2017)</span></span></span><!--[if supportFields]><span style='mso-bookmark:
_Hlk72500077'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Recent studies have shown that screening
on a regular basis can help detect any traces of CRC and these growths can be
removed before they turn life threatening </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72500077'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;URL&quot;:&quot;https://www.cancercare.org/publications/116-colorectal_cancer_the_importance_of_screening_and_early_detection&quot;,&quot;accessed&quot;:{&quot;date-parts&quot;:[[&quot;2021&quot;,&quot;5&quot;,&quot;17&quot;]]},&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;0&quot;]]},&quot;title&quot;:&quot;Colorectal
Cancer | Early Detection and Screening | CancerCare&quot;,&quot;type&quot;:&quot;webpage&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=f1f41866-af85-3faa-91e1-209d03feea4c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Colorectal
Cancer | Early Detection and Screening | CancerCare n.d.)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Colorectal
Cancer | Early Detection and Screening | CancerCare
n.d.)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Colorectal Cancer |
Early Detection and Screening | CancerCare
n.d.)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Colorectal
Cancer | Early Detection and Screening | CancerCare n.d.)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. One of the methods for detecting
mutations in CRC is allele-specific PCR. This has the ability to detect APS and
KRAS mutations similar to the Sanger sequencing method. Sanger sequencing
focuses on the area of interest and narrows down the nucleotide sequences. It
can even detect novel changes but since the run time and cost is high, this
method is not considered for routine checks. Allele- specific PCR amplifies the
mutational sequence by using primers. This increases speed and accuracy and is
therefore preferred over Sanger sequencing </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72500077'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/1471-2407-10-660&quot;,&quot;ISSN&quot;:&quot;14712407&quot;,&quot;PMID&quot;:&quot;21122130&quot;,&quot;abstract&quot;:&quot;Background:
Epidermal growth factor receptor inhibitor therapy is now approved for
treatment of metastatic colorectal carcinomas (CRC) in patients with tumors lacking
KRAS mutations. Several procedures to detect KRAS mutations have been
developed. However, the analytical sensitivity and specificity of these assays
on routine clinical samples are not yet fully characterised.Methods: The
practical aspects and clinical applicability of a KRAS-assay based on Pyrosequencing
were evaluated in a series of 314 consecutive CRC cases submitted for
diagnostic KRAS analysis. The performance of Pyrosequencing compared to
allele-specific, real-time PCR was then explored by a direct comparison of
CE-IVD-marked versions of Pyrosequencing and TheraScreen (DxS) KRAS assays for
a consecutive subset (n = 100) of the 314 clinical CRC samples.Results: Using
Pyrosequencing, 39% of the 314 CRC samples were found KRAS-mutated and several
of the mutations (8%) were located in codon 61. To explore the analytical
sensitivity of the Pyrosequencing assay, mutated patient DNA was serially
diluted with wild-type patient DNA. Dilutions corresponding to 1.25-2.5% tumor
cells still revealed detectable mutation signals. In clinical practice, our
algorithm for KRAS analysis includes a reanalysis of samples with low tumor
cell content (&lt; 10%, n = 56) using an independent assay (allele-specific
PCR, DxS). All mutations identified by Pyrosequencing were then confirmed and,
in addition, one more mutated sample was identified in this subset of 56
samples. Finally, a direct comparison of the two technologies was done by
re-analysis of a subset (n = 100) of the clinical samples using CE-IVD-marked
versions of Pyrosequencing and TheraScreen KRAS assays in a single blinded
fashion. The number of samples for which the KRAS codon 12/13 mutation status
could be defined using the Pyrosequencing or the TheraScreen assay was 94 and
91, respectively, and both assays detected the same number of codon 12 and 13
mutations.Conclusions: KRAS mutation detection using Pyrosequencing was
evaluated on a consecutive set of clinical CRC samples. Pyrosequencing provided
sufficient analytical sensitivity and specificity to assess the mutation status
in routine formalin-fixed CRC samples, even in tissues with a low tumor cell
content.  2010 Sundstrm et al; licensee BioMed Central
Ltd.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Sundstrm&quot;,&quot;given&quot;:&quot;Magnus&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Edlund&quot;,&quot;given&quot;:&quot;Karolina&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lindell&quot;,&quot;given&quot;:&quot;Monica&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Glimelius&quot;,&quot;given&quot;:&quot;Bengt&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Birgisson&quot;,&quot;given&quot;:&quot;Helgi&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Micke&quot;,&quot;given&quot;:&quot;Patrick&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Botling&quot;,&quot;given&quot;:&quot;Johan&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;BMC
Cancer&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2010&quot;,&quot;12&quot;,&quot;1&quot;]]},&quot;publisher&quot;:&quot;BMC
Cancer&quot;,&quot;title&quot;:&quot;KRAS analysis in colorectal carcinoma:
Analytical aspects of Pyrosequencing and allele-specific PCR in clinical
practice&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=0a831f42-b72d-319f-a333-186cdee24f1c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Sundstrm
et al. 2010)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Sundstrm et
al. 2010)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Sundstrm et al.
2010)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Sundstrm
et al. 2010)</span></span></span><!--[if supportFields]><span style='mso-bookmark:
_Hlk72500077'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. Real-time PCR makes use of a fluorogenic
probe which is exclusive to the target sequence. Change in the fluorescent
signals is used to monitor target DNA molecules. This takes less time than Sanger
sequencing and allele-specific PCR as the turnover time is less </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72500077'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1016/j.humpath.2012.03.003&quot;,&quot;ISSN&quot;:&quot;00468177&quot;,&quot;PMID&quot;:&quot;22658275&quot;,&quot;abstract&quot;:&quot;There
has been an increasing role for molecular diagnostics in the diagnosis and
management of cancer, and colorectal carcinoma is no exception. Recent
molecular advances have elucidated 3 broad molecular subtypes of colorectal
cancer, including chromosomal instability, microsatellite instability, and
cytosine-phosphoguanine island methylator phenotype, which will be discussed.
Also, the common syndromes associated with colorectal carcinoma will be
reviewed with a focus on the differentiation between Lynch syndrome and microsatellite
unstable tumors. Molecular biomarkers for predictive and prognostic markers are
also becoming widely used, and due to the clinical use of monoclonal antibodies
to the epidermal growth factor receptor, an emphasis is placed on that pathway.
 2012 Elsevier Inc. All rights
reserved.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Legolvan&quot;,&quot;given&quot;:&quot;Mark
P.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Taliano&quot;,&quot;given&quot;:&quot;Ross
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Resnick&quot;,&quot;given&quot;:&quot;Murray
B.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Human
Pathology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2012&quot;,&quot;8&quot;]]},&quot;page&quot;:&quot;1157-1168&quot;,&quot;publisher&quot;:&quot;Hum
Pathol&quot;,&quot;title&quot;:&quot;Application of molecular techniques in the
diagnosis, prognosis and management of patients with colorectal cancer: A
practical approach&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;43&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=fade6b8d-8313-3e99-bf27-47ad37374197&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Legolvan,
Taliano, and Resnick
2012)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Legolvan, Taliano,
and Resnick 2012)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Legolvan,
Taliano, and Resnick 2012)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72500077'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72500077'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>.</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-spacerun:yes'></span><span style='mso-no-proof:yes'><span
style='mso-spacerun:yes'> </span></span></span><b><span
style='font-size:14.0pt;line-height:150%;font-family:"Times New Roman",serif'>B.
Differential Gene Expression of colorectal cancer genes</span></b><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-no-proof:yes'><o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Differential gene
expression was done using Galaxy software and various tools available within. Quality
control is a necessary step before proceeding with mapping and <span
class=SpellE>MultiQC</span>, a tool to create a single report visualising
output from multiple tools across many samples was used.</span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1093/bioinformatics/btw354&quot;,&quot;ISSN&quot;:&quot;14602059&quot;,&quot;PMID&quot;:&quot;27312411&quot;,&quot;abstract&quot;:&quot;Motivation:
Fast and accurate quality control is essential for studies involving next-generation
sequencing data. Whilst numerous tools exist to quantify QC metrics, there is
no common approach to flexibly integrate these across tools and large sample
sets. Assessing analysis results across an entire project can be time consuming
and error prone; batch effects and outlier samples can easily be missed in the
early stages of analysis. Results: We present MultiQC, a tool to create a
single report visualising output from multiple tools across many samples,
enabling global trends and biases to be quickly identified. MultiQC can plot
data from many common bioinformatics tools and is built to allow easy extension
and customization. Availability and implementation: MultiQC is available with
an GNU GPLv3 license on GitHub, the Python Package Index and Bioconda.
Documentation and example reports are available at
http://multiqc.info.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ewels&quot;,&quot;given&quot;:&quot;Philip&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Magnusson&quot;,&quot;given&quot;:&quot;Mns&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lundin&quot;,&quot;given&quot;:&quot;Sverker&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kller&quot;,&quot;given&quot;:&quot;Max&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2016&quot;,&quot;10&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;3047-3048&quot;,&quot;publisher&quot;:&quot;Oxford
University Press&quot;,&quot;title&quot;:&quot;MultiQC: Summarize analysis
results for multiple tools and samples in a single
report&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;32&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=b1c51ee6-7300-36e7-b573-c63887468baa&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Ewels
et al. 2016)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Ewels et al.
2016)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Ewels et al.
2016)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(<span class=SpellE>Ewels</span> et al. 2016)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> In a study done
by Alexander <span class=SpellE>Dobin</span> and Thomas R. <span class=SpellE>Gingerasm</span>
</span><!--[if supportFields]><span style='font-size:12.0pt;line-height:150%;
font-family:"Times New Roman",serif'><span style='mso-element:field-begin;
mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1002/0471250953.bi1114s51&quot;,&quot;ISSN&quot;:&quot;1934340X&quot;,&quot;PMID&quot;:&quot;26334920&quot;,&quot;abstract&quot;:&quot;Mapping
of large sets of high-throughput sequencing reads to a reference genome is one
of the foundational steps in RNA-seq data analysis. The STAR software package
performs this task with high levels of accuracy and speed. In addition to
detecting annotated and novel splice junctions, STAR is capable of discovering
more complex RNA sequence arrangements, such as chimeric and circular RNA. STAR
can align spliced sequences of any length with moderate error rates, providing
scalability for emerging sequencing technologies. STAR generates output files
that can be used for many downstream analyses such as transcript/gene
expression quantification, differential gene expression, novel isoform
reconstruction, and signal visualization. In this unit, we describe
computational protocols that produce various output files, use different RNA-seq
datatypes, and utilize different mapping strategies. STAR is open source
software that can be run on Unix, Linux, or Mac OS X systems.  2015 by John
Wiley &amp; Sons,
Inc.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dobin&quot;,&quot;given&quot;:&quot;Alexander&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Gingeras&quot;,&quot;given&quot;:&quot;Thomas
R.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Current
Protocols in
Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2015&quot;,&quot;9&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;11.14.1-11.14.19&quot;,&quot;publisher&quot;:&quot;John
Wiley and Sons Inc.&quot;,&quot;title&quot;:&quot;Mapping RNA-seq Reads with
STAR&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;51&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=b7193bf1-338f-36cd-ae3d-c273e207fa92&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Dobin
and Gingeras
2015)&quot;,&quot;manualFormatting&quot;:&quot;(2015)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Dobin
and Gingeras 2015)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Dobin
and Gingeras 2015)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(2015)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> <a
name="_Hlk72189691">it was found that the STAR software package performed the
task of mapping with high speed and accuracy. It was also found that STAR was
useful for generating files which could be used for downstream analysis such as
differential gene expression, gene expression quantification, novel isoform
reconstruction, etc. For further analysis, it was important to know the number
of reads that were mapped to each of the genomic features, i.e., to each gene
or exon. This process known as summarisation is required for genomic analysis.
Data obtained from STAR was analysed using </a><span class=SpellE><span
style='mso-bookmark:_Hlk72189691'>FeatureCounts</span></span><span
style='mso-bookmark:_Hlk72189691'> developed by Yang Liao ,Gordon Smyth and Wei
Shi </span></span><!--[if supportFields]><span style='mso-bookmark:_Hlk72189691'></span><span
style='mso-element:field-begin;mso-field-lock:yes'></span><span
style='mso-bookmark:_Hlk72189691'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>ADDIN CSL_CITATION {&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1093/bioinformatics/btt656&quot;,&quot;ISSN&quot;:&quot;14602059&quot;,&quot;PMID&quot;:&quot;24227677&quot;,&quot;abstract&quot;:&quot;Motivation:
Next-generation sequencing technologies generate millions of short sequence
reads, which are usually aligned to a reference genome. In many applications,
the key information required for downstream analysis is the number of reads
mapping to each genomic feature, for example to each exon or each gene. The
process of counting reads is called read summarization. Read summarization is
required for a great variety of genomic analyses but has so far received
relatively little attention in the literature.Results: We present
featureCounts, a read summarization program suitable for counting reads
generated from either RNA or genomic DNA sequencing experiments. featureCounts
implements highly efficient chromosome hashing and feature blocking techniques.
It is considerably faster than existing methods (by an order of magnitude for
gene-level summarization) and requires far less computer memory. It works with
either single or paired-end reads and provides a wide range of options
appropriate for different sequencing applications.  2013 The Author 2013.
Published by Oxford University Press. All rights
reserved.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Liao&quot;,&quot;given&quot;:&quot;Yang&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Smyth&quot;,&quot;given&quot;:&quot;Gordon
K.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shi&quot;,&quot;given&quot;:&quot;Wei&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2014&quot;,&quot;4&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;923-930&quot;,&quot;publisher&quot;:&quot;Oxford
University Press&quot;,&quot;title&quot;:&quot;FeatureCounts: An efficient
general purpose program for assigning sequence reads to genomic
features&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;30&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=a180e1b3-4094-32e8-9a56-6dd530dedeb6&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Liao,
Smyth, and Shi
2014)&quot;,&quot;manualFormatting&quot;:&quot;(2014)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Liao,
Smyth, and Shi 2014)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Liao,
Smyth, and Shi 2014)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72189691'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(2014)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72189691'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72189691'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. They proposed that <span
class=SpellE>featureCounts</span> used highly effective feature blocking
technique and chromosome hashing techniques. It was proved to be much faster
that the already existing techniques and utilised far comparatively lesser
computer memory. It works with single ended as well as pair ended reads and
provides a range of options for various sequencing options. <span
style='mso-spacerun:yes'></span>Crucial step in analysis of the data obtained
from RNA-<span class=SpellE>seq</span>, count data, is the detection of the
differentially expressed genes.<span style='mso-spacerun:yes'> </span>A
question arises regarding the quantification and interference of systematic
changes between provided conditions as compared to that within condition
variability. Theis was answered by M. I. Love, W. Huber, S. Anders when they came
up with the DESeq2 package which provide methods for testing the estimates of
dispersion, generalised linear models and logarithmic fold change to <span
class=SpellE>in corporate</span> the data driven prior distributions </span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72189691'></span><span style='mso-element:field-begin;
mso-field-lock:yes'></span><span style='mso-bookmark:_Hlk72189691'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/s13059-014-0550-8&quot;,&quot;abstract&quot;:&quot;A
basic task in the analysis of count data from RNA-seq is the detection of
differentially expressed genes. The count data are presented as a table which
reports, for each sample, the number of sequence fragments that have been
assigned to each gene. Analogous data also arise for other assay types,
including comparative ChIP-Seq, HiC, shRNA screening, mass spectrometry. An
important analysis question is the quantification and statistical inference of systematic
changes between conditions, as compared to within-condition variability. The
package DESeq2 provides methods to test for differential expression by use of
negative binomial generalized linear models; the estimates of dispersion and
logarithmic fold changes incorporate data-driven prior distributions 1. This
vignette explains the use of the package and demonstrates typical workflows. An
RNA-seq workflow 2 on the Bioconductor website covers similar material to this
vignette but at a slower pace, including the generation of count matrices from
FASTQ files. DESeq2 version: 1.12.3 If you use DESeq2 in published research,
please cite: M. I. Love, W. Huber, S. Anders: Moderated estimation of fold
change and dispersion for RNA-seq data with DESeq2. Genome Biology 2014,
15:550.
http://dx.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Love&quot;,&quot;given&quot;:&quot;Michael
I&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Anders&quot;,&quot;given&quot;:&quot;Simon&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Huber&quot;,&quot;given&quot;:&quot;Wolfgang&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;researchgate.net&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2016&quot;]]},&quot;title&quot;:&quot;Differential
analysis of count data-the DESeq2
package&quot;,&quot;type&quot;:&quot;article-journal&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=3c98543d-fcff-3c34-b98c-67e9de8af0fd&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Love,
Anders, and Huber 2016)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Love,
Anders, and Huber
2016)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Love, Anders, and
Huber
2016)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></span><![endif]--><span
style='mso-bookmark:_Hlk72189691'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'><span style='mso-no-proof:yes'>(Love,
Anders, and Huber 2016)</span></span></span><!--[if supportFields]><span
style='mso-bookmark:_Hlk72189691'></span><span style='mso-element:field-end'></span><![endif]--><span
style='mso-bookmark:_Hlk72189691'><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>. <span
style='mso-spacerun:yes'></span>This paper focused on the changes made to
DESeq2 from its earlier version <span class=SpellE>DESeq</span>. The new DESeq2
package uses weighted quantile function of <span class=SpellE>Hmisc</span>
package instead of the earlier methods of matching the empirical quantile to
the normal distribution quantile for the calculation of beta prior variance (October
2014). The paper published in 2014 mentioned that <span class=SpellE>DESeq</span>
used log fold shrinkage to only applied interaction terms however in a change
implemented in 2015, DESeq2 was programmed to turn off the log fold shrinkage
for all terms if interaction terms were present.</span></span><span
style='mso-bookmark:_Hlk72189691'></span><span style='font-size:13.5pt;
line-height:150%;font-family:"Georgia",serif;color:#2E2E2E'> <o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif;color:#2E2E2E'>Volcano
plots are graphical tools which are used for analysis of levels of mRNA
expression. This data is mainly obtained via microarray technology </span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'><span style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN
CSL_CITATION {&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/gb-2003-4-4-210&quot;,&quot;ISSN&quot;:&quot;14656906&quot;,&quot;PMID&quot;:&quot;12702200&quot;,&quot;abstract&quot;:&quot;Extracting
biological information from microarray data requires appropriate statistical
methods. The simplest statistical method for detecting differential expression
is the t test, which can be used to compare two conditions when there is
replication of samples. With more than two conditions, analysis of variance
(ANOVA) can be used, and the mixed ANOVA model is a general and powerful
approach for microarray experiments with multiple factors and/or several
sources of
variation.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Cui&quot;,&quot;given&quot;:&quot;Xiangqin&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Churchill&quot;,&quot;given&quot;:&quot;Gary
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Genome
Biology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2003&quot;]]},&quot;title&quot;:&quot;Statistical
tests for differential expression in cDNA microarray
experiments&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;4&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=38c6b6c9-d553-3063-bd12-609470057400&quot;]},{&quot;id&quot;:&quot;ITEM-2&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/ng766&quot;,&quot;abstract&quot;:&quot;Here
we present a statistically rigorous approach to quantifying microarray
expression data that allows the relative effects of multiple classes of
treatment to be compared and incorporates analytical methods that are common to
quantitative genetics. From the magnitude of gene effects and contributions of
variance components, we find that gene expression in adult flies is affected
most strongly by sex, less so by genotype and only weakly by age (for 1-and
6-wk flies); in addition, sex  genotype interactions may be present for as
much as 10% of the Drosophila transcriptome. This interpretation is compromised
to some extent by statistical issues relating to power and experimental design.
Nevertheless, we show that changes in expression as small as 1.2-fold can be
highly significant. Genotypic contributions to transcriptional variance may be of
a similar magnitude to those relating to some quantitative phenotypes and
should be considered when assessing the significance of
experi&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Wolfinger&quot;,&quot;given&quot;:&quot;Russ&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Jin&quot;,&quot;given&quot;:&quot;Wei&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Riley&quot;,&quot;given&quot;:&quot;Rebecca
M&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Wolfinger&quot;,&quot;given&quot;:&quot;Russell
D&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;White&quot;,&quot;given&quot;:&quot;Kevin
P&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Passador-Gurgel&quot;,&quot;given&quot;:&quot;Gisele&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Gibson&quot;,&quot;given&quot;:&quot;Greg&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;nature.com&quot;,&quot;id&quot;:&quot;ITEM-2&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2001&quot;]]},&quot;title&quot;:&quot;MAQC
peoject View project phase analysis of cell cycle genes in cell culture and
circadian gene expression in tissues View project The contributions of sex,
genotype and age to transcriptional variance in Drosophila melanogaster mental
treatments&quot;,&quot;type&quot;:&quot;article-journal&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=70ac1933-38ef-3950-b87e-3dc4cb0d91f8&quot;]},{&quot;id&quot;:&quot;ITEM-3&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1142/S0219720012310038&quot;,&quot;abstract&quot;:&quot;A
volcano plot displays unstandardized signal (e.g. log-fold-change) against
noise-adjusted/standardized signal (e.g. t-statistic or -log 10(p-value) from
the t-test). We review the basic and interactive use of the volcano plot and
its crucial role in understanding the regularized t-statistic. The joint
filtering gene selection criterion based on regularized statistics has a curved
discriminant line in the volcano plot, as compared to the two perpendicular
lines for the \&quot;double filtering\&quot; criterion. This review attempts to
provide a unifying framework for discussions on alternative measures of
differential expression, improved methods for estimating variance, and visual
display of a microarray analysis result. We also discuss the possibility of
applying volcano plots to other fields beyond microarray.  2012 Imperial
College
Press.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;biology&quot;,&quot;given&quot;:&quot;W
Li - Journal of bioinformatics and computational&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;2012&quot;,&quot;given&quot;:&quot;undefined&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;World
Scientific&quot;,&quot;id&quot;:&quot;ITEM-3&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,&quot;12&quot;]]},&quot;title&quot;:&quot;Application
of Volcano Plots in Analyses of mRNA Differential Expressions with
Microarrays&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=68452501-49c0-3985-8272-4ef97917c234&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(biology
and 2012 2011; Cui and Churchill 2003; Wolfinger et al.
2001)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(biology and 2012
2011; Cui and Churchill 2003; Wolfinger et al.
2001)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(biology and 2012
2011; Cui and Churchill 2003; Wolfinger et al.
2001)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'><span style='mso-no-proof:yes'>(biology and 2012 2011; Cui and
Churchill 2003; Wolfinger et al. 2001)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'><span style='mso-element:field-end'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'>. Volcano plots are a type of scatter plots where each point on
the plot represents a gene or a probe set. The x-coordinate is the (log)
fold-change (FC) and the y-coordinate is the t-statistic or &#8722; log 10 of
the p-value from a t-test. Volcano plots are preferred for microarray analysis
due to its ability to simultaneously display two, correlated pieces of
information, i.e., fold change and t-statistics. Major advantage of volcano
plot is that it addresses joint gene filtering </span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'><span style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN
CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/1471-2105-10-402&quot;,&quot;ISSN&quot;:&quot;14712105&quot;,&quot;PMID&quot;:&quot;19995439&quot;,&quot;abstract&quot;:&quot;Background:
Many researchers use the double filtering procedure with fold change and t test
to identify differentially expressed genes, in the hope that the double
filtering will provide extra confidence in the results. Due to its simplicity,
the double filtering procedure has been popular with applied researchers
despite the development of more sophisticated methods. Results: This paper, for
the first time to our knowledge, provides theoretical insight on the drawback
of the double filtering procedure. We show that fold change assumes all genes
to have a common variance while t statistic assumes gene-specific variances. The
two statistics are based on contradicting assumptions. Under the assumption
that gene variances arise from a mixture of a common variance and gene-specific
variances, we develop the theoretically most powerful likelihood ratio test
statistic. We further demonstrate that the posterior inference based on a
Bayesian mixture model and the widely used significance analysis of microarrays
(SAM) statistic are better approximations to the likelihood ratio test than the
double filtering procedure. Conclusion: We demonstrate through hypothesis
testing theory, simulation studies and real data examples, that well
constructed shrinkage testing methods, which can be united under the mixture
gene variance assumption, can considerably outperform the double filtering
procedure.  2009 Zhang and Cao; licensee BioMed Central
Ltd.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Zhang&quot;,&quot;given&quot;:&quot;Song&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Cao&quot;,&quot;given&quot;:&quot;Jing&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;BMC
Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2009&quot;,&quot;12&quot;,&quot;8&quot;]]},&quot;title&quot;:&quot;A
close examination of double filtering with fold change and t test in microarray
analysis&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=f159fd25-3af6-3075-8837-abe0fe2889f3&quot;]},{&quot;id&quot;:&quot;ITEM-2&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1142/S0219720012310038&quot;,&quot;abstract&quot;:&quot;A
volcano plot displays unstandardized signal (e.g. log-fold-change) against
noise-adjusted/standardized signal (e.g. t-statistic or -log 10(p-value) from
the t-test). We review the basic and interactive use of the volcano plot and
its crucial role in understanding the regularized t-statistic. The joint
filtering gene selection criterion based on regularized statistics has a curved
discriminant line in the volcano plot, as compared to the two perpendicular
lines for the \&quot;double filtering\&quot; criterion. This review attempts to
provide a unifying framework for discussions on alternative measures of
differential expression, improved methods for estimating variance, and visual
display of a microarray analysis result. We also discuss the possibility of
applying volcano plots to other fields beyond microarray.  2012 Imperial
College Press.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;biology&quot;,&quot;given&quot;:&quot;W
Li - Journal of bioinformatics and
computational&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;2012&quot;,&quot;given&quot;:&quot;undefined&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;World
Scientific&quot;,&quot;id&quot;:&quot;ITEM-2&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,&quot;12&quot;]]},&quot;title&quot;:&quot;Application
of Volcano Plots in Analyses of mRNA Differential Expressions with
Microarrays&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=a4da0e3d-8578-3fc8-915f-15b378e52d65&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(biology
and 2012 2011; Zhang and Cao
2009)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(biology and 2012
2011; Zhang and Cao
2009)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(biology and 2012
2011; Zhang and Cao
2009)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
color:#2E2E2E'><span style='mso-no-proof:yes'>(biology and 2012 2011; Zhang and
Cao 2009)</span></span><!--[if supportFields]><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif;color:#2E2E2E'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif;color:#2E2E2E'>
where both log FC and t- statistic is used to measure differential expressions.
<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'>C. Molecular Signatures<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Signalling pathways
represented as mechanistic models lay out a bridge from the variations in
phenotype to the variations in the genotype. This is done at the scale of gene
integrity or activity. These bridges are built over graphs and represent the
knowledge of compound functional relationships among the proteins present
within the cells as described in major repositories namely <span class=SpellE>Reactome</span>,
KEGG, or other more specialised and complex such as Disease Maps. They provide
a theoretical framework for analysis of genomic variation or gene expression
and their effect over phenotypic responses like cell death and proliferation
and the downstream processes involved which are needed for understanding drug
response and progression of the disease. These mechanistic models have given
insight into various disease mechanisms behind rare diseases, diabetes, cancers
including neuroblastoma and glioblastoma, obesity. <o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span class=SpellE><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>PathLinker</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
is a signalling pathway reconstruction method used for constructing protein
interactions and regulatory networks. It mainly connects the receptors in a
given query to the transcription factors. Its main use is to connect sources to
any given targets in an interaction network. The working algorithm of <span
class=SpellE>PathLinker</span> is computation of the k-shortest and simplest
paths in a given network from a particular source to a target. It incorporates
Yens algorithm which allows effective computation of k values. The output is a
sub-network composed of the k shortest paths.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'>D. Variant Calling<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>The process of
identifying variants from sequence datasets is known as variant calling. It is
done by aligning sequence to the reference genome and creating BAM files. When
analysing germline variant calling, the reference genome for the species of
interest is standard. This aids in the identification of genotypes. Since most
of the genomes are diploid, at any given locus, all reads either have same base
which indicates homozygosity or half of these reads will have one base and the
other half have another base thus indicating heterozygosity. However, in
somatic variant calling, a related tissue from the same individual is taken as
the reference. Mosaicism can be expected to between cells.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:300%'><span
style='font-size:12.0pt;line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:300%'><span
style='font-size:12.0pt;line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:300%'><span
style='font-size:12.0pt;line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:300%'><span
style='font-size:12.0pt;line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><span style='font-size:12.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:300%'><b><span style='font-size:16.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='line-height:300%'><b><span style='font-size:16.0pt;
line-height:300%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='line-height:300%'><b><span style='font-size:16.0pt;
line-height:300%;font-family:"Times New Roman",serif'>Methods and Materials<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:200%'><span style='font-size:14.0pt;
line-height:200%;font-family:"Times New Roman",serif'>RNA Sequence Data<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>The <span class=SpellE>Fastq</span>
files for RNA sequence analysis were obtained online from GEO (Gene Expression
Omnibus). 4 diseased and 4 adjacent controls for colorectal cancer were taken
from the GEO series GSE50760. All the datasets obtained were pair ended.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:200%'><span style='font-size:14.0pt;
line-height:200%;font-family:"Times New Roman",serif'>Differential Gene
Expression of RNA Sequence Data<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>In this part differentially
expressed genes were detected. The following is the procedure used to detect
the DE genes and find out the genes of significance and in turn generate a
volcano plot for visualisation.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
datasets were uploaded online and analysed using the GALAXY tool. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>FastQC</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
was used to produce a report stating the quality of the sequences and <span
class=SpellE>MultiQC</span> </span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1093/bioinformatics/btw354&quot;,&quot;ISSN&quot;:&quot;14602059&quot;,&quot;PMID&quot;:&quot;27312411&quot;,&quot;abstract&quot;:&quot;Motivation:
Fast and accurate quality control is essential for studies involving
next-generation sequencing data. Whilst numerous tools exist to quantify QC
metrics, there is no common approach to flexibly integrate these across tools
and large sample sets. Assessing analysis results across an entire project can
be time consuming and error prone; batch effects and outlier samples can easily
be missed in the early stages of analysis. Results: We present MultiQC, a tool
to create a single report visualising output from multiple tools across many
samples, enabling global trends and biases to be quickly identified. MultiQC
can plot data from many common bioinformatics tools and is built to allow easy
extension and customization. Availability and implementation: MultiQC is
available with an GNU GPLv3 license on GitHub, the Python Package Index and
Bioconda. Documentation and example reports are available at
http://multiqc.info.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ewels&quot;,&quot;given&quot;:&quot;Philip&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Magnusson&quot;,&quot;given&quot;:&quot;Mns&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lundin&quot;,&quot;given&quot;:&quot;Sverker&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Kller&quot;,&quot;given&quot;:&quot;Max&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2016&quot;,&quot;10&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;3047-3048&quot;,&quot;publisher&quot;:&quot;Oxford
University Press&quot;,&quot;title&quot;:&quot;MultiQC: Summarize analysis
results for multiple tools and samples in a single
report&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;32&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=b1c51ee6-7300-36e7-b573-c63887468baa&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Ewels
et al. 2016)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Ewels et al.
2016)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Ewels et al.
2016)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(Ewels et al. 2016)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> was used to combine
the reports generated by <span class=SpellE>FastQC</span>.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Cutadapt</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'> </span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.14806/ej.17.1.200&quot;,&quot;abstract&quot;:&quot;When
small RNA is sequenced on current sequencing machines, the resulting reads are
usually longer than the RNA and therefore contain parts of the 3' adapter. That
adapter must be found and removed error-tolerantly from each read before read
mapping. Previous solutions are either hard to use or do not offer required
features, in particular support for color space data. As an easy to use
alternative, we developed the command-line tool cutadapt, which supports 454,
Illumina and SOLiD (color space) data, offers two adapter trimming algorithms,
and has other useful features.<span style='mso-spacerun:yes'>
</span>Cutadapt, including its MIT-licensed source code, is available for
download at<span style='mso-spacerun:yes'>
</span>http://code.google.com/p/cutadapt/&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Martin&quot;,&quot;given&quot;:&quot;Marcel&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;EMBnet.journal&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,&quot;5&quot;,&quot;2&quot;]]},&quot;page&quot;:&quot;10&quot;,&quot;publisher&quot;:&quot;EMBnet
Stichting&quot;,&quot;title&quot;:&quot;Cutadapt removes adapter sequences from
high-throughput sequencing reads&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;17&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=fff29488-c50c-3e87-83e2-f97a1fd7d977&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Martin
2011)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Martin
2011)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Martin 2011)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(Martin 2011)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> was used to
further improve the quality by trimming. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Gene
annotation file was obtained from the UCSC Main server with the <span
class=SpellE>ncbiRefSeq</span> form of annotation. RNA Star </span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1093/bioinformatics/bts635&quot;,&quot;ISSN&quot;:&quot;13674803&quot;,&quot;PMID&quot;:&quot;23104886&quot;,&quot;abstract&quot;:&quot;Motivation:
Accurate alignment of high-throughput RNA-seq data is a challenging and yet
unsolved problem because of the non-contiguous transcript structure, relatively
short read lengths and constantly increasing throughput of the sequencing
technologies. Currently available RNA-seq aligners suffer from high mapping
error rates, low mapping speed, read length limitation and mapping
biases.Results: To align our large (&gt;80 billon reads) ENCODE Transcriptome
RNA-seq dataset, we developed the Spliced Transcripts Alignment to a Reference
(STAR) software based on a previously undescribed RNA-seq alignment algorithm
that uses sequential maximum mappable seed search in uncompressed suffix arrays
followed by seed clustering and stitching procedure. STAR outperforms other
aligners by a factor of &gt;50 in mapping speed, aligning to the human genome
550 million 2  76 bp paired-end reads per hour on a modest 12-core server,
while at the same time improving alignment sensitivity and precision. In
addition to unbiased de novo detection of canonical junctions, STAR can
discover non-canonical splices and chimeric (fusion) transcripts, and is also
capable of mapping full-length RNA sequences. Using Roche 454 sequencing of
reverse transcription polymerase chain reaction amplicons, we experimentally
validated 1960 novel intergenic splice junctions with an 80-90% success rate,
corroborating the high precision of the STAR mapping strategy.  The Author(s)
2012. Published by Oxford University
Press.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dobin&quot;,&quot;given&quot;:&quot;Alexander&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Davis&quot;,&quot;given&quot;:&quot;Carrie
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Schlesinger&quot;,&quot;given&quot;:&quot;Felix&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Drenkow&quot;,&quot;given&quot;:&quot;Jorg&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Zaleski&quot;,&quot;given&quot;:&quot;Chris&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Jha&quot;,&quot;given&quot;:&quot;Sonali&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Batut&quot;,&quot;given&quot;:&quot;Philippe&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Chaisson&quot;,&quot;given&quot;:&quot;Mark&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Gingeras&quot;,&quot;given&quot;:&quot;Thomas
R.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2013&quot;,&quot;1&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;15-21&quot;,&quot;publisher&quot;:&quot;Oxford
Academic&quot;,&quot;title&quot;:&quot;STAR: Ultrafast universal RNA-seq
aligner&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;29&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=5bd827e2-60ae-30ce-be87-5a9dd2730dde&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Dobin
et al. 2013)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Dobin et al.
2013)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Dobin et al.
2013)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(Dobin et al. 2013)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> was used to align
the RNA-<span class=SpellE>seq</span> reads to the reference genome. The
reference genome used was hg38. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Featurecounts</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
was used on the obtained report to count the number of genes per annotated
gene.</span> <span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>DESeq2
</span><!--[if supportFields]><span style='font-size:12.0pt;line-height:150%;
font-family:"Times New Roman",serif'><span style='mso-element:field-begin;
mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/s13059-014-0550-8&quot;,&quot;abstract&quot;:&quot;A
basic task in the analysis of count data from RNA-seq is the detection of
differentially expressed genes. The count data are presented as a table which
reports, for each sample, the number of sequence fragments that have been
assigned to each gene. Analogous data also arise for other assay types,
including comparative ChIP-Seq, HiC, shRNA screening, mass spectrometry. An
important analysis question is the quantification and statistical inference of
systematic changes between conditions, as compared to within-condition variability.
The package DESeq2 provides methods to test for differential expression by use
of negative binomial generalized linear models; the estimates of dispersion and
logarithmic fold changes incorporate data-driven prior distributions 1. This
vignette explains the use of the package and demonstrates typical workflows. An
RNA-seq workflow 2 on the Bioconductor website covers similar material to this
vignette but at a slower pace, including the generation of count matrices from
FASTQ files. DESeq2 version: 1.12.3 If you use DESeq2 in published research,
please cite: M. I. Love, W. Huber, S. Anders: Moderated estimation of fold
change and dispersion for RNA-seq data with DESeq2. Genome Biology 2014,
15:550. http://dx.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Love&quot;,&quot;given&quot;:&quot;Michael
I&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Anders&quot;,&quot;given&quot;:&quot;Simon&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Huber&quot;,&quot;given&quot;:&quot;Wolfgang&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;researchgate.net&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2016&quot;]]},&quot;title&quot;:&quot;Differential
analysis of count data-the DESeq2
package&quot;,&quot;type&quot;:&quot;article-journal&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=3c98543d-fcff-3c34-b98c-67e9de8af0fd&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Love,
Anders, and Huber
2016)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Love, Anders, and
Huber 2016)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Love, Anders,
and Huber
2016)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-no-proof:yes'>(Love, Anders, and Huber 2016)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'> was used to normalise
and analyse the obtained count data and produce a combined table.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>In
order to extract the genes with a significant change in gene expression, i.e.,
adjusted p-value below 0.05, filter data on any column using simple expression
function was used. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Further
the genes with absolute fold change (FC) &gt; 2 were selected. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
obtained file was further annotated to find the gene names, functions and the
location of the gene within the genome. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>DeSeq2
file was used to generate the volcano plot for visualisation of upregulated and
down regulated genes.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>DeSeq2
file was used to segregate the up regulated and the down regulated genes
according to its expression level. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:18.0pt;mso-add-space:auto;
text-indent:-18.0pt;line-height:150%;mso-list:l2 level1 lfo1'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
gene IDs of the regulated genes were submitted to an ID conversion and gene set
analysis tool (<span class=SpellE>SynGo</span>) in order to find the entrez
gene number, name and function of the gene, <span class=SpellE>taxid</span> and
symbol, alias and HGNC.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:115%'><span style='font-size:14.0pt;
line-height:115%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:200%'><span style='font-size:14.0pt;
line-height:200%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:200%'><span style='font-size:14.0pt;
line-height:200%;font-family:"Times New Roman",serif'>Obtaining Molecular
Signatures<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Clustering was done using
2 types of analysis; clustering analysis and shortest path analysis. This was
done using the software <span class=SpellE>Cytoscape</span>. Packages such as <span
class=SpellE>GeneMania</span>, MCODE and <span class=SpellE>PathLinker</span>
were installed.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
genes were separated as up-regulated and down-regulated. Gene Symbols of the
upregulated genes were selected and uploaded to <span class=SpellE>GeneMania</span>,
thereby creating a network of all the upregulated genes.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>MCODE
was used to analyse the network formed and seed nodes for the network were
isolated.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
first ten most upregulated genes were taken and its position in the network was
noted.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>PathLinker</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
was used for the ten selected genes and the seed genes and a cluster was
generated showing shortest path.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:18.0pt;mso-add-space:auto;
text-indent:-18.0pt;line-height:150%;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
same was done for downregulated genes and another cluster was obtained.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:200%'><span style='font-size:14.0pt;
line-height:200%;font-family:"Times New Roman",serif'>Variant Calling<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Direct comparison of data
is needed between data obtained from normal and tumor tissue samples in order
to differentiate between these two types of variants. This process was
completed using Galaxy. Following is the process used for variant calling.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
same four diseased datasets and four controls were used. Quality control was
done using <span class=SpellE>FastQC</span> and <span class=SpellE>MultiQC</span>.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Trimmomatic</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'> was
used to trim and filter the normal tissue reads as well as tumor reads.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>The
reads from normal tissue sample and tumor samples were mapped to reference
genome using Map with BWA-MEM tool. Reference genome used here is hg19.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Filter
BAM datasets on a variety of attribute tool was used to produce BAM dataset
with reads which were successfully mapped to the reference genome. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>RmDup</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
was used to remove duplicate reads from normal as well as tumor datasets.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>BamLeftAlign</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
was used to left-align reads around indels. Read mapping qualities were
recalibrated using <span class=SpellE>CalMD</span>. Filter BAM datasets on a
variety of attributes function was used to remove reads where mapping quality
was set to 255 by <span class=SpellE>CalMD</span>.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
class=SpellE><span style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>VarScan</span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>
somatic was used for classifying variants into germline, somatic and LOH event variants.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Variant
and gene annotation files were uploaded onto galaxy namely</span> <span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>variant
annotations from Cancer Hotspots, the Cancer Biomarkers database of the Cancer
Genome Interpreter (CGI) project, the <span class=SpellE>CIViC</span> database,
variant annotations from <span class=SpellE>dbSNP</span> and lists of genes
annotated with the keywords proto-oncogene or tumor suppressor at <span
class=SpellE>UniProt</span>.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Genomic
annotations were added to dataset of variants with <span class=SpellE>SnpEff</span>.
Clinical and genetic evidence-based annotations were added using GEMINI.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>GEMINI
annotate was used to add annotations to the variants in an already loaded
GEMINI database. It was also used to add information from <span class=SpellE>dbSNP</span>,
cancer hotspots and</span> <span style='font-size:12.0pt;line-height:150%;
font-family:"Times New Roman",serif'>Cancer Genome Interpreter (CGI) and links
to the <span class=SpellE>CIViC</span> database.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:18.0pt;mso-add-space:
auto;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>GEMINI
query was used for querying database for somatic variants and for generating
reports of genes affected by variants.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:18.0pt;mso-add-space:auto;
text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo3'><![if !supportLists]><span
style='font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:
Symbol;mso-bidi-font-family:Symbol'><span style='mso-list:Ignore'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Information
about <span class=SpellE>UniProt</span> cancer genes, CGI biomarkers and <span
class=SpellE>CIViC</span> was further added to the report and columns were
arranged accordingly.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:115%'><span style='font-size:14.0pt;
line-height:115%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:115%'><span style='font-size:14.0pt;
line-height:115%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:115%'><span style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:12.0pt;line-height:200%;font-family:"Times New Roman",serif;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-ansi-language:
EN-IN;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'><br clear=all
style='page-break-before:always'>
</span>

<p class=MsoNormal align=center style='text-align:center;line-height:200%'><b><span
style='font-size:16.0pt;line-height:200%;font-family:"Times New Roman",serif'>RESULTS<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Differential Gene
Expression <span class=SpellE>Resul</span><o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Alignment result of RNA
STAR showing percentage of reads mapped to the reference genome. The above
figure shows the alignment percentages of tumor tissue, whereas the one below
shows the alignment percentage of adjacent normal tissue. In both cases, the
alignment percentage for genes being mapped uniquely is higher than 80% for all
the samples. The reference genome used was hg38. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>MA
plot showing the gene expression change resulting from two conditions: tumor
tissue and adjacent normal tissue<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>The plot is constructed
between M, which is the expression change in conditions, and A, the average mean
also known as average expression strength of the genes. Significant genes are
marked in red. The red line represents M=0. Those points clustered near this
line are the genes which do not have a change in their expression. The ones away
from the lines are those which are regulated. Genes above the M=0 line <span
class=GramE>are</span> up-regulated whereas those below are down regulated.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-spacerun:yes'> </span><span style='mso-spacerun:yes'></span><span
style='mso-spacerun:yes'></span><o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-family:"Times New Roman",serif'>Figure:
Volcano plot showing up and downregulated genes</span><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-spacerun:yes'> </span></span><span style='font-family:"Times New Roman",serif'>Figure:
Volcan plot showing first 250 significant genes<span
style='mso-spacerun:yes'> </span><span
style='mso-spacerun:yes'></span></span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;line-height:107%;font-family:
"Times New Roman",serif'>Volcano plots showing the differentially expressed
genes. The points on the graph marked in blue are the downregulated genes and
those marked in red are the up-regulated genes. Genes which are not significant
are grey in colour.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;line-height:107%;font-family:
"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;line-height:107%;font-family:
"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Table:
Top ten highly up-regulated genes<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Table:
Top ten highly downregulated genes<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Tables
showing the top ten up-regulated and downregulated genes obtained through differential
gene analysis<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Molecular Signatures</span></b><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:14.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Figure:
Molecular Signature obtained by analysing up-regulated genes. The genes in
yellow are the seed genes, those in blue are the highly up-regulated genes and
the rest are the moderately up-regulated genes. This shows the shortest paths
between the highly up-regulated genes and the seed genes.<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Figure:
Molecular Signature obtained by analysing downregulated genes. The genes in
yellow are the seed genes, those in blue are the highly downregulated genes and
the rest are the moderately downregulated genes. This shows the shortest paths
between the highly downregulated genes and the seed genes<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><b><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Variant Calling<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Following are the results
of genes affected due to somatic variations. The tables provide information
about the gene which is affected, the <span class=SpellE>hgnc</span> ID, Entrez
ID and the synonym of the gene affected. Additionally, it gives Residual
Variation Intolerance Score (RVIS) percentage which helps interpret sequence
data. OG and TS stand for oncogene and tumor suppressor gene and 1 <span
class=GramE>stands</span> for positive result.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><a name="_Hlk72370178"><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Genes
affected due to somatic variation in dataset SRR975569. 72 genes were found to
be affected by the variants</span></a><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><a name="_Hlk72370632"><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Genes
affected due to somatic variation in dataset SRR975570. 52 genes were found to
be affected by the variants.</span></a><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><a name="_Hlk72370825"><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>Genes
affected due to somatic variation in dataset SRR975571. 58 genes were found to
be affected by the variants</span></a><span style='font-size:12.0pt;line-height:
150%;font-family:"Times New Roman",serif'>.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'>Genes affected due to
somatic variation in dataset <a name="_Hlk72397375">SRR975572</a>. 58 genes
were found to be affected by the variants.<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><b><span
style='font-size:16.0pt;line-height:150%;font-family:"Times New Roman",serif'>Discussion<o:p></o:p></span></b></p>

<p class=MsoNormal style='line-height:150%'><a name="_Hlk72400618"><span
lang=EN-US style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:
bold'>This study was to done to build a transcriptome profile for CRC and
identify the key regulatory genes involved</span></a><span lang=EN-US
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:
bold'>. Transcriptome contains the entire messenger RNA (mRNA) in an
individual. It contains the genetic code that is transcribed into RNA
molecules. In this study eight datasets were analyzed, four tumor tissue
samples and four adjacent normal unaffected tissue samples. Datasets were
checked for quality and trimmed accordingly. RNA STAR was used the map datasets
onto a reference genome. Reference genome used in this study was hg38. Upon
mapping, STAR produced a BAM file containing information about the sequence
quality and location of reads on the chromosome and the mapping quality. Upon
quality check, <span class=SpellE>MultiQC</span> report showed that more than
80% of the reads were uniquely aligned to the reference genome. These were the
finding for all the datasets taken. These results were viewed using </span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:bold'>Integrative
Genomics Viewer </span><!--[if supportFields]><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'><span style='mso-element:field-begin;
mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/nbt.1754&quot;,&quot;ISSN&quot;:&quot;10870156&quot;,&quot;PMID&quot;:&quot;21221095&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Robinson&quot;,&quot;given&quot;:&quot;James
T.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Thorvaldsdttir&quot;,&quot;given&quot;:&quot;Helga&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Winckler&quot;,&quot;given&quot;:&quot;Wendy&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Guttman&quot;,&quot;given&quot;:&quot;Mitchell&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lander&quot;,&quot;given&quot;:&quot;Eric
S.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Getz&quot;,&quot;given&quot;:&quot;Gad&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Mesirov&quot;,&quot;given&quot;:&quot;Jill
P.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Biotechnology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,&quot;1&quot;,&quot;1&quot;]]},&quot;page&quot;:&quot;24-26&quot;,&quot;publisher&quot;:&quot;Nature
Publishing Group&quot;,&quot;title&quot;:&quot;Integrative genomics
viewer&quot;,&quot;type&quot;:&quot;article&quot;,&quot;volume&quot;:&quot;29&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=722827ce-1652-33fd-ad79-1c06c981596d&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Robinson
et al. 2011)&quot;,&quot;manualFormatting&quot;:&quot;(IGV, Robinson et al.
2011)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Robinson et al.
2011)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Robinson et al.
2011)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]--><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:bold'><span
style='mso-no-proof:yes'>(IGV, Robinson et al. 2011)</span></span><!--[if supportFields]><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:bold'><span
style='mso-element:field-end'></span></span><![endif]--><span style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'>. A counts table was generated with
the number of fragments mapped to a particular gene. This table was used to as
an input for DESeq2 which is a tool for sorting RNA sequence data and running
differential gene analysis. Two factors were taken: tumor and normal. Three main
results were generated as a result of DESeq2. The first was a table which
contained the normalised counts. The second was a graphical summary containing
five components. The first was a PCA plot generated on the account of
normalised counts, the next was a heatmap highlighting the similarities and
dissimilarities between the samples. The third and fourth were dispersion
estimates and a histogram of p values for genes in comparison. The fifth and
the main component was the MA plot. Working theory for the plot is that most
genes do not experience a change in their expression and should be plotted at 0
on the y-axis because log (1) is 0. The genes which passed a significance
threshold value of 0.1 (p- value &lt; 0.1) were marked in red. Those genes
lying above M=0 are the up-regulated genes and those below are the
downregulated genes. The last result generated by DESeq2 was a file which
contained following values for every gene: identification, Wald statistic, fold
change and standard error in log2, p-value and the adjusted p-value. Genes with
a significant change (adjusted p-value &lt; 0.05) were extracted. 631 genes
were found to be significantly expressed. In order to check for the up and
downregulated, a volcano plot was plotted. DESeq2 file containing values for
each gene was used. Significant threshold was kept at 0.05. The results showed
a plot with </span><a name="_Hlk72401488"><span lang=EN-US style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:bold'>469
up-regulated genes and 162 downregulated genes</span></a><span lang=EN-US
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:
bold'>. <i>SLC35D3, STC2, COMP, CYP2B6, BOK-AS1, FOXQ1, CEL, EMSLR </i>and<i>
CCAT1 </i>are the top ten upregulated genes. <i>AK1, KRTAP13-2, ZNF492, B3GALT1,
RETNLB, MT1M, B3GALT5-AS1, B3GNT7, WFDC2 </i>and <i>HIGD1A </i>are the top ten
downregulated genes. Genes were sorted based on their regulation and they were
used to produce density-based clusters. Two networks were built: a network
comprising of up-regulated genes and a network comprising of down regulated
genes. MCODE analysis produced a list of seeds. Seeds are the highly connected
genes in a cluster. Significant genes affect the seed and as a result the genes
interacting with the seed are affected as well. From the cluster of up-regulated
genes 11 seeds were identified. <span class=SpellE>PathLinker</span> was used
to calculate the shortest path. <span class=SpellE>PathLinker</span> needs
sources (significant genes) and targets (seeds) to calculate the path. The 11
seed genes were treated as targets and the top 10 up-regulated genes were used
to create molecular signature highlighting the shortest path between the seeds
and genes. a total of 50 shortest possible paths were generated. 13 paths were
identified with a path score of 1 and the rest had a path score of 2. The path
ranked first was between <i>ADAMTS2</i> (significant gene) and <i>IGFBP5</i>
(seed). Similar method was followed for downregulated genes. From the cluster
of downregulated genes, 5 seeds were identified. Top ten downregulated genes
were used to create molecular signatures. This resulted in the creation of 50
shortest paths. 8 paths were identified with a path score of 1 and the rest had
a path score 2. The shortest path which was ranked first was between <i>LGALS3</i>
(significant gene) and <i>MT1HL1</i> (seed). Variant calling is the process of
identifying variants from the sequence datasets. In this study, somatic and
germline variants were identified from tumor and normal sample datasets.
Datasets were checked for quality and trimmed and mapped to reference genome. <span
class=SpellE>VarScan</span> somatic was used to determine the variant present
in tumor and normal pair and classify variants into germline, somatic and LOH
variants. GEMINI was used to annotate variants and extract information from <span
class=SpellE>VarScan</span> results. Annotations from various cancer databases
were added to GEMINI in order to create a detailed report. Essentially at this
point a report with somatic status, cancer hotspot information, links for <span
class=SpellE>CIVic</span> sites and CGI data is created. In dataset SRR</span><span
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'>975569,
total number of variants identified were 54915 (germline: 50416, somatic: 2772,
LOH: 1712). In dataset SRR975570, total number of variants identified were
34145 (germline: 30539, somatic: 2331, LOH: 1264). In dataset SRR975571, total
number of variants identified were 47640 (germline: 45729, somatic: 1713, LOH:
197). In dataset SRR975572, total number of variants identified were 48991
(germline: 44514, somatic: 2784, LOH: 1677). </span><span lang=EN-US
style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:
bold'>However, a detailed report is further created where only somatic
variations are considered. Somatic variants with p-value &lt; 0.05 were
considered. Tables provided in the study mention the gene affected by the
variant and other important features such as nature of gene (tumor suppressor
gene or oncogene), chromosome number and entrez number. In dataset SRR975569,
72 genes affected by somatic variants were identified; in SRR975570, 52 genes
were identified; in SRR975571, 58 genes were identified and in SRR975572, 58
genes, which were affected by somatic variants, were identified.<o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span lang=EN-US style='font-size:
12.0pt;line-height:150%;font-family:"Times New Roman",serif;mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-bidi-font-weight:bold'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:150%'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;font-family:"Times New Roman",serif'><span
style='mso-spacerun:yes'></span></span><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:12.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;
line-height:150%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%;font-family:
"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<span style='font-size:14.0pt;line-height:107%;font-family:"Times New Roman",serif;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-ansi-language:
EN-IN;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'><br clear=all
style='mso-special-character:line-break;page-break-before:always'>
</span>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%;font-family:
"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Times New Roman",serif'>Conclusion<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=EN-US style='font-size:12.0pt;line-height:107%;
font-family:"Times New Roman",serif;mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-bidi-font-weight:bold'>This study was to done to
build a transcriptome profile for CRC and identify the key regulatory genes
involved. Eight mRNA datasets were analyzed and RNA sequencing was done in
order to get the differentially expressed genes. When data was mapped to a
reference genome after quality checks and appropriate trimming, about 80% of
the reads were mapped uniquely. Counts table was generated for the purpose of
finding out the number of fragments mapped and subsequently obtain the
differentially obtained genes. DESeq2 was used to generate a file with information
needed to plot a volcano plot. Based on this plot, up-regulated and down
regulated genes were visualized. 469 up-regulated genes and 162 downregulated
genes were obtained. These genes served as a basis for creating and visualizing
molecular signatures and eventually finding the shortest path between
significant genes and the seed genes. 50 shortest paths were obtained for
up-regulated as well as downregulated clusters. Variant calling was performed with
tumor and normal data pairs. Different variations (somatic, germline and LOH
event mutations) were found in every dataset. On finer refinement, tables
containing only somatic variations which passed p-value &lt; 0.05 were allowed
and the genes affected by these variations were highlighted</span></p>

</div>

</body>

</html>
